src/wfs/src/test/java/org/geoserver/wfs/TransactionTest.java

```diff
@@ -438,4 +439,94 @@ public void testUpdateWithDifferentPrefix() throws Exception {
               .getFirstChild().getNodeValue());
    }
 
+    @Test
+    public void testInsert11Features() throws Exception {
+        String features = "<wfs:Transaction service=\"WFS\" version=\"1.0.0\" "
+                + "xmlns:cgf=\"http://www.opengis.net/cite/geometry\" "
+                + "xmlns:ogc=\"http://www.opengis.net/ogc\" "
+                + "xmlns:wfs=\"http://www.opengis.net/wfs\" "
+                + "xmlns:gml=\"http://www.opengis.net/gml\"> "
+                + "<wfs:Insert > " + "<cgf:Lines>" + "<cgf:lineStringProperty>"
```

Comment:
Consider sending in your own id values:

```
<cfg:Lines id="t002">
```

And then we can see if they are preserved by the WFS Transaction - that will only work if the datastore on the backend can preserve ids.